src/nrc/datamodel/admin.py

```diff
@@ -135,3 +152,23 @@ def created_date(self, obj):
         return timezone.make_aware(naive, timezone=timezone.utc)
 
     created_date.short_description = _("Created date")
+
+    def get_inline_instances(self, request, obj=None):
+        # Hide the NotificatieResponseInline when creating a Notification
+        return obj and super().get_inline_instances(request, obj) or []
+
+    def save_model(self, request, obj, form, change):
+        """
+        Given a model instance save it to the database.
+        """
+        super().save_model(request, obj, form, change)
+
+        # Notification is being resent, delete the existing notificatieresponses
+        if change:
+            obj.notificatieresponse_set.all().delete()
```

Comment:
I changed this by adding a field `attempt` to `NotificatieResponse` and filtering on the latest attempts in the admin (https://github.com/open-zaak/open-notificaties/pull/63/files#diff-d77d5ac8c678284a8344153644fb0232c040223d10ed6e55dbd84deee2cc08dfR128, surely this can be simplified? :sweat_smile:)